ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -51,7 +51,7 @@ define([
             propertyJSON.type = 'LOCATION';
         }
 
-        propertyJSON.placeholder = propertyJSON.type === 'DATE' ? 'DD MMM YYYY HH:mm:ss.SSS' : 'Use * for wildcard.';
```

Comment:
Might be worth moving the placeholder into the core date component so everywhere will have this placeholder.